tests/models/test_lm_eval_simple.py

```diff
@@ -0,0 +1,85 @@
+import huggingface_hub
+import pytest
+import transformers
+
+from tests.models.test_checkpoint import _prepare_resume_fn
+from tests.utils.model_configs import ModelTestingGroup
+from tests.utils.utils import requires_cuda, requires_lm_eval
+
+# NOTE: These tests only verify that the functionality runs without crashing.
+# NOTE: The tokenizer is from a LLaMA-style model, which may not be suitable for all models,
+#       but it should be sufficient since we are not concerned with actual accuracy in this tests.
+
+
+@pytest.fixture(scope="module")
+def model_path(result_path):
+    return huggingface_hub.snapshot_download(
+        repo_id="HuggingFaceTB/SmolLM2-135M-Instruct",
+        local_dir=result_path / "lm_eval/model",
+    )
+
+
+def get_lm_eval_config(base_path, tokenizer_path):
+    tokenizer = transformers.AutoTokenizer.from_pretrained(tokenizer_path)
+    return [
+        f"data.tokenizer.path={tokenizer_path}",
+        f"model.base_model.vocab_size={tokenizer.vocab_size}",
+        "training.evaluators.evaluation_test.interval=1",
+        "training.evaluators.evaluation_test.evaluator.type=lm_eval",
+        "training.evaluators.evaluation_test.evaluator.cli_args="
+        f'["--tasks","gsm8k,xnli_en,wikitext","--output_path","{str(base_path / "lm_eval")}","--limit","10"]',
+    ]
+
+
+@pytest.mark.extra_slow
```

Comment:
How long does this take? It would be worrying not to have any tests other than extra-slow.